packages/patternfly-4/react-core/src/experimental/components/Drawer/__snapshots__/Drawer.test.tsx.snap

```diff
@@ -4,13 +4,9 @@ exports[`Drawer isExpanded = false and isInline = false 1`] = `
 <div
   className="pf-c-drawer"
 >
-  <div
-    className="pf-c-drawer__content"
-  >
-     
+  <Component>
```

Comment:
Should change test to use mount instead of shallow so we can get more info here

Reply:
Sure thing!